libs/types/src/lib/api/lems/graphql/judging.graphql

```diff
@@ -58,6 +58,9 @@ type Judging {
   "Room IDs available for judging in this division"
   rooms: [String!]!
 
+  "All rooms in this division (for judge advisor view)"
+  allRooms: [Room!]!
```

Comment:
Please use division.rooms for this
(I will push the dev tools this week and then itll be easier to discover the schema)